common/djangoapps/student/views.py

```diff
@@ -1754,6 +1755,9 @@ def create_account_with_params(request, params):
             }
         )
 
+    # Announce registration
+    REGISTER_USER.send(sender=None, user=user, profile=profile)
```

Comment:
I'm surprised that the `created` field in `post_save` wasn't enough. Are we both saving and then updating the `UserProfile` during registration?